sympy/printing/tests/test_str.py

```diff
@@ -660,14 +660,14 @@ def test_settings():
 def test_RandomDomain():
     from sympy.stats import Normal, Die, Exponential, pspace, where
     X = Normal('x1', 0, 1)
-    assert str(where(X > 0)) == "Domain: And(0 < x1, x1 < oo)"
+    assert str(where(X > 0)) == "Domain: (0 < x1 & x1 < oo)"
 
     D = Die('d1', 6)
-    assert str(where(D > 4)) == "Domain: Or(Eq(d1, 5), Eq(d1, 6))"
+    assert str(where(D > 4)) == "Domain: (Eq(d1, 5) | Eq(d1, 6))"
 
     A = Exponential('a', 1)
     B = Exponential('b', 1)
-    assert str(pspace(Tuple(A, B)).domain) == "Domain: And(0 <= a, 0 <= b, a < oo, b < oo)"
+    assert str(pspace(Tuple(A, B)).domain) == "Domain: (0 <= a & 0 <= b & a < oo & b < oo)"
```

Comment:
Honestly, I find the behavior in the example above odd. I would expect the following results to be equal:
```python
>>> srepr(x < y & z)
"StrictLessThan(Symbol(′x′),And(Symbol(′y′),Symbol(′z′)))"

>>> srepr((x < y) & z)
"And(Symbol(′z′),StrictLessThan(Symbol(′x′),Symbol(′y′)))"
```
As for the python reference, it agrees with what I was trying to say in my previous answer.
```python
>>> 2 > 2 and True
False

>>> (2 > 2) and True
False

>>> 2 > (2 and True)
True
```